dashboard/app/controllers/api/v1/regional_partners_controller.rb

```diff
@@ -92,9 +92,9 @@ def get_partner_cohort_capacity(regional_partner_value, role)
       partner_id = regional_partner_value ? regional_partner_value : current_user.regional_partners.first
       regional_partner = RegionalPartner.find_by(id: partner_id)
       if role == 'csd_teachers'
-        return regional_partner.cohort_capacity_csd
+        return regional_partner&.cohort_capacity_csd
       elsif role == 'csp_teachers'
-        return regional_partner.cohort_capacity_csp
+        return regional_partner&.cohort_capacity_csp
```

Comment:
should this be `0` rather than `nil` in this case?

Reply:
I think `nil` is more appropriate in this case where there is no regional partner (as opposed to a partner with a 0 capacity), especially given that this method already returns `nil` in another code path: https://github.com/code-dot-org/code-dot-org/blob/306e6c96103ae285358094286c6434b55eeb3894/dashboard/app/controllers/api/v1/regional_partners_controller.rb#L100

And the client handles `null` capacity by hiding the cohort calculator: https://github.com/code-dot-org/code-dot-org/blob/76e8d7ea38a374b61695ec114a778e896dc69e50/apps/src/code-studio/pd/application_dashboard/cohort_calculator.jsx#L71-L73